setup.py

```diff
@@ -1,7 +1,7 @@
 import os
 from setuptools import find_packages, setup
 
-install_requires = ['mls', 'scikit-learn']
+install_requires = ['scikit-learn', 'mls @ git+https://github.com/ratschlab/renku-mls@master']
```

Comment:
scikit-learn could be dropped, and lets keep the mls as it's already released on testpypi and i'll release this week on pypi

Reply:
reason for scikit: _extract_mls has the import statement, so the load is dynamic and only done if you receive a sklearn model; it's safe to say then that you will have sklearn in the env

Reply:
> scikit-learn could be dropped, and lets keep the mls as it's already released on testpypi and i'll release this week on pypi

yeah but https://pypi.org/project/mls/

Reply:
note that
 a) https://test.pypi.org/project/mls/
 b) "i'll release this week on pypi"

Reply:
under a different name? sorry; not familiar with how this works